src/contracts/interfaces/IEigenPod.sol

```diff
@@ -38,6 +37,13 @@ interface IEigenPod {
         uint64 partialWithdrawalAmountGwei;
     }
 
+    struct ValidatorInfo {
+        uint64 validatorIndex;
+        uint64 restakedBalanceGwei;
+        uint32 lastWithdrawalBlockNumber;
+        VALIDATOR_STATUS status;
+    }
```

Comment:
can we document what all of these are for / intended usage? Am curious in particular about `lastWithdrawalBlockNumber`

Reply:
this is some mistake, removed it.